exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/Otel2PrometheusConverterTest.java

```diff
@@ -0,0 +1,257 @@
+
+    // Uncomment to debug exposition format output
+    // System.out.println(expositionFormat);
+
+    Matcher matcher = PATTERN.matcher(expositionFormat);
+    assertThat(matcher.matches()).isTrue();
+    assertThat(matcher.group("help")).isEqualTo(expectedHelp);
+    assertThat(matcher.group("type")).isEqualTo(expectedType);
+    // Note: Summaries and histograms produce output which matches METRIC_NAME_PATTERN multiple
+    // times. The pattern ends up matching against the first.
+    assertThat(matcher.group("metricName")).isEqualTo(expectedMetricName);
+  }
+
+  private static Stream<Arguments> metricMetadataArgs() {
+    // TODO (jack-berg): delete "Previously metricName was .." comments before merging, update
+    // comments to reflect new logic
+    return Stream.of(
+        // special case for gauge
+        // Previously metricName was "sample_ratio"
+        Arguments.of(
+            createSampleMetricData("sample", "1", MetricDataType.LONG_GAUGE),
+            "sample_ratio gauge",
+            "sample_ratio description",
+            "sample_ratio"),
+        // special case for gauge with drop - metric unit should match "1" to be converted to
+        // "ratio"
+        // Previously metricName was "sample"
+        Arguments.of(
+            createSampleMetricData("sample", "1{dropped}", MetricDataType.LONG_GAUGE),
+            "sample_ratio gauge",
+            "sample_ratio description",
+            "sample_ratio"),
+        // Gauge without "1" as unit
+        // Previously metricName was "sample_unit"
+        Arguments.of(
+            createSampleMetricData("sample", "unit", MetricDataType.LONG_GAUGE),
+            "sample_unit gauge",
+            "sample_unit description",
+            "sample_unit"),
+        // special case with counter
+        // Previously metricName was "sample_unit_total"
+        Arguments.of(
+            createSampleMetricData("sample", "unit", MetricDataType.LONG_SUM),
+            "sample_unit_total counter",
+            "sample_unit_total description",
+            "sample_unit_total"),
+        // special case unit "1", but not gauge - "1" is dropped
+        // Previously metricName was "sample_total"
+        Arguments.of(
+            createSampleMetricData("sample", "1", MetricDataType.LONG_SUM),
+            "sample_ratio_total counter",
+            "sample_ratio_total description",
+            "sample_ratio_total"),
+        // units expressed as numbers other than 1 are retained
+        // Previously metricName was "sample_2_total"
+        Arguments.of(
+            createSampleMetricData("sample", "2", MetricDataType.LONG_SUM),
+            "sample_2_total counter",
+            "sample_2_total description",
+            "sample_2_total"),
+        // metric name with unsupported characters
+        // Previously metricName was "s_ple_percent_per_minute"
+        Arguments.of(
+            createSampleMetricData("s%%ple", "%/min", MetricDataType.SUMMARY),
```

Comment:
`m` used to convert to `minute` but now converts to `meter`. Updated from `%/m` to `%/min` to keep the assertions the same.